examples/30_extended/datasets_tutorial.py

```diff
@@ -108,3 +109,39 @@
     alpha=0.8,
     cmap="plasma",
 )
+
+
+############################################################################
+# Edit a created dataset
+# =================================================
+# Use test server
+openml.config.start_using_configuration_for_example()
+############################################################################
+# Changes to these field edits existing version
```

Comment:
Add something to the effect of "A dataset may only be edited by their original uploader."